tests/providers/google/common/hooks/test_base_google.py

```diff
@@ -632,6 +643,63 @@ def test_authorize_assert_http_timeout_is_present(self, mock_get_credentials):
         http_authorized = self.instance._authorize().http
         self.assertNotEqual(http_authorized.timeout, None)
 
+    @mock.patch(MODULE_NAME + '.get_credentials_and_project_id')
+    def test_get_credentials_and_project_id_with_impersonation_chain_as_string(
+        self,
+        mock_get_creds_and_proj_id
+    ):
+        mock_credentials = mock.MagicMock()
+        mock_get_creds_and_proj_id.return_value = (mock_credentials, "PROJECT_ID")
+        self.instance.impersonation_chain = "ACCOUNT_1"
+        result = self.instance._get_credentials_and_project_id()
+        mock_get_creds_and_proj_id.assert_called_once_with(
+            key_path=None,
+            keyfile_dict=None,
+            scopes=self.instance.scopes,
+            delegate_to=None,
+            target_principal="ACCOUNT_1",
+            delegates=None,
+        )
+        self.assertEqual((mock_credentials, "PROJECT_ID"), result)
+
+    @mock.patch(MODULE_NAME + '.get_credentials_and_project_id')
+    def test_get_credentials_and_project_id_with_single_account_in_impersonation_chain(
+        self,
+        mock_get_creds_and_proj_id
+    ):
+        mock_credentials = mock.MagicMock()
+        mock_get_creds_and_proj_id.return_value = (mock_credentials, "PROJECT_ID")
+        self.instance.impersonation_chain = ["ACCOUNT_1"]
```

Comment:
Should we use parametrize for this value? Both tests seem very similar

Reply:
Yes, I think we can parametrize even all three new tests from this class. I will change that.